src/style.css

```diff
@@ -1 +1,52 @@
-/* add styles here */
+html {
+  margin: 0;
+  padding: 0;
+}
+
+.body {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 140px;
+  height: 160px;
+  padding: 22px;
+}
+
+.stars {
+  display: inline-flex;
+}
+
+.stars:nth-child(n + 2) {
+  margin-top: 4px;
+}
+
+.stars__star {
```

Comment:
Don't use tag names for styling (except 'html' and 'body'). It's better to style all elements using classes. The class '.stars__star' is good, but ensure that no tag selectors are used elsewhere in your CSS.